style.css

```diff
@@ -0,0 +1,670 @@
+* {
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+  text-decoration: none;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+nav {
+  background-color: #fff;
+  border-radius: 5px;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 50px;
+  padding-top: 20px;
```

Comment:
- [ ] I would suggest adding a `padding right` of `50px` to your `navbar`, but to avoid repeating  the key word `padding`, you do this 👉️  `padding: 20px 50px 0 50px`. Please would you mind doing that?